ts/editor/rich-text-input/RichTextInput.svelte

```diff
@@ -72,7 +72,9 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
 
     async function moveCaretToEnd(): Promise<void> {
         const richText = await richTextPromise;
-        placeCaretAfterContent(richText);
+        if (richText.textContent) {
+            placeCaretAfterContent(richText);
+        }
```

Comment:
This looks good, but it will be hard to understand it later on.
How about something like this:

```typescript
if (richText.textContent.length === 0) {
    // your explanation why this is necessary
    return;
}

placeCaretAfterContent(richText);
```

Reply:
Revised, is this ok?

Reply:
Looks good - thank you for the fix!